rabbitmq/src/tests/utils_rmqtest.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <userver/utest/utest.hpp>
+
+#include <userver/clients/dns/resolver.hpp>
+#include <userver/rabbitmq.hpp>
+
+USERVER_NAMESPACE_BEGIN
+
+class ClientWrapper final {
```

Comment:
Usage snippets and docs would be appreciated

Reply:
Yea, that's in my todo list